cmd/namespace/create.go

```diff
@@ -29,15 +29,17 @@ import (
 
 // CreateOptions represents the options that namespace create has
 type CreateOptions struct {
-	Members   *[]string
-	Namespace string
-	Show      bool
+	Members      *[]string
+	Namespace    string
+	Show         bool
+	SetCurrentNs bool
 }
 
 // Create creates a namespace
 func Create(ctx context.Context) *cobra.Command {
 	options := &CreateOptions{
-		Show: true,
+		Show:         false,
+		SetCurrentNs: false,
```

Comment:
```suggestion
		SetCurrentNs: false,
```

I'm assuming that `SetCurrentNs` value will be taken as `false` when we pass no flag , something like `okteto namespace create <ns-name>`. However, No matter what, the value is automatically taken as true same as default value for flag. Any idea why? cc @jLopezbarb

Reply:
Fixed it!